module/documents/chat-message.mjs

```diff
@@ -415,10 +415,10 @@ export default class ChatMessage5e extends ChatMessage {
         <li data-uuid="${uuid}" class="target ${isMiss ? "miss" : "hit"}">
           <i class="fas ${isMiss ? "fa-times" : "fa-check"}"></i>
           <div class="name">${name}</div>
-          ${ac ? `
+          ${(ac !== "") ? `
           <div class="ac">
             <i class="fas fa-shield-halved"></i>
-            <span>${ac}</span>
+            <span>${(ac === null) ? "<i class='fa-solid fa-infinity'></i>" : ac}</span>
```

Comment:
Using the infinity letter rather than the font awesome would probably look more correctly sized:

<img width="296" alt="Screenshot 2024-10-02 at 10 03 17" src="https://github.com/user-attachments/assets/7a188e08-d8eb-48be-825c-759133be0e1b">
<img width="297" alt="Screenshot 2024-10-02 at 10 03 07" src="https://github.com/user-attachments/assets/e039509c-03e1-4e96-bb4a-ff5534d7c420">


```suggestion
            <span>${(ac === null) ? "&infin;" : ac}</span>
```